src/ChargebeeError.ts

```diff
@@ -0,0 +1,14 @@
+export interface ChargebeeError {
+  code: string;
+  message: string;
+  domain: string;
```

Comment:
Why do we need `domain` and `userInfo` as part of the error object?

Reply:
Not necessarily. But these are the common properties received at the JS layer from the native exception handlers.